### DISCUSSION TITLE
Potential issues with raw memory allocation

### DISCUSSION THREAD

**Discussion by ericjack371** [cpp-for-yourself/lectures-and-homeworks#106] [Issues withe lectures] [answered]
In C++, what are some potential issues when using raw pointers for dynamic memory allocation, and how can these issues be mitigated?

**Comment by AFN4NX** [comment]
Memory Leaks
Dangling Pointers
Double Deletion

**Comment by niosus** [comment, marked as answer]
Thanks for the question. I think it is very relevant. @AFN4NX already gave some hints on what to look for but I also did a video on stuff very related to that here: https://youtu.be/eHcdTytDZrI?si=IWFIyVfzMDURwJ5W

Somewhere in the second part of the video we talk about these potential problems. 

Please have a look and don't hesitate to ask more questions if that video does not cover your needs.

**Reply by ericjack371** [reply]
thank you very much
